curvefs/src/client/error_code.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/*
+ * Project: curve
+ * Created Date: Thur May 27 2021
+ * Author: xuchaojie
+ */
+
+#include "curvefs/src/client/error_code.h"
+
+namespace curvefs {
+namespace client {
+
+using ::curvefs::metaserver::MetaStatusCode;
+
+CURVEFS_ERROR MetaStatusCodeToCurvefsErrCode(
+    MetaStatusCode code) {
+    CURVEFS_ERROR ret = CURVEFS_ERROR::UNKNOWN;
+    switch (code) {
+        case MetaStatusCode::OK:
```

Comment:
MetaStatusCode::RPC_ERROR shuold be handled

Reply:
done

Reply:
done